internal/acceptance/service_principal_test.go

```diff
@@ -14,6 +21,76 @@ const awsSpn = `resource "databricks_service_principal" "this" {
 	display_name = "SPN {var.RANDOM}"
 }`
 
+func TestAccServicePrincipalHomeDeleteSuccess(t *testing.T) {
+	appId := "12345a67-8b9c-0d1e-23fa-4567b89cde04"
+	appId2 := "22345a67-8b9c-0d1e-23fa-4567b89cde04"
+	os.Setenv("appId", appId)
```

Comment:
in the case of this particular test, use string concatenation instead of `env.appId`. integration tests are run in parallel and environment variables are not isolated per test.

also, environment variables are usually ALL_CAPS.